release/downloadIstioCandidate.sh

```diff
@@ -34,10 +34,21 @@ fi
 NAME="istio-$ISTIO_VERSION"
 URL="https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-${OSEXT}.tar.gz"
 echo "Downloading $NAME from $URL ..."
-curl -L "$URL" | tar xz
-# TODO: change this so the version is in the tgz/directory name (users trying multiple versions)
-echo "Downloaded into $NAME:"
-ls "$NAME"
-BINDIR="$(cd "$NAME/bin" && pwd)"
-echo "Add $BINDIR to your path; e.g copy paste in your shell and/or ~/.profile:"
-echo "export PATH=\"\$PATH:$BINDIR\""
+curl -L "$URL" | tar xz 
+echo ""
+echo "Istio ${ISTIO_VERSION} Download Complete!"
+echo ""
+echo "Istio has been successfully downloaded into the ${NAME} folder on your system. 🎉\n"
+echo ""
+BINDIR="$(cd "${NAME}/bin" && pwd)"
+echo "Next Steps:\n"
+echo "To add Istio to a Kubernetes cluster, see https://istio.io/docs/setup/kubernetes/#evaluating-istio"
```

Comment:
I would prefer not to point to the evaluating istio link.

Reply:
Probably 🎉 is not the best character for a text message on all platforms.

Reply:
@linsun https://istio.io/docs/setup/kubernetes/install/kubernetes/  <-- would yo prefer this link instead?

@esnible I'll just take it out. I was directly copying what I saw in linkerd.

Reply:
how about https://istio.io/docs/setup/kubernetes/install/?   the link you pointed to is our quick start which is the demo profile.  @frankbu mentioned today that some people think demo profile is the only thing we support to get started and we should avoid that.

Reply:
@linsun ok, makes sense. thanks for the info.